webserver/views/api/v1/core.py

```diff
@@ -109,3 +111,20 @@ def _validate_offset(offset):
     else:
         offset = 0
     return offset
+
+@bp_core.route("/<user_api_key>/get_pending_jobs", methods=["GET"])
```

Comment:
In the API we have a way of logging in using the api key as a token (https://github.com/metabrainz/acousticbrainz-server/blob/f62ffa81c82f81f8c91d2816dffcfeb9ca098790/webserver/login/__init__.py#L38)
Instead of adding the user key as a parameter here, you can make the API endpoint require a logged in user and then use the user id in the pending jobs query